proto/bpfd.proto

```diff
@@ -15,13 +15,15 @@ message BytecodeImage {
     string password = 4;
 }
 
+message NoLocation{}
+
 message LoadRequestCommon {
     oneof location {
         BytecodeImage image = 1;
         string file = 2;
     } ;
     string section_name = 3;
-    int32 program_type = 4;
+    uint32 program_type = 4;
```

Comment:
This just mirrors the kernel type from https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L6301

 Keeping the types the same between our `Load` and `List` APIs makes keeps it simple to reconcile in our k8s setup, otherwise type munging is required